Makefile

```diff
@@ -152,6 +152,41 @@ install-pipelines: ## Install the CNBI pipelines into the configured cluster.
 uninstall-pipelines: ## Uninstall the CNBI pipelines from  the configured cluster. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	kubectl delete --ignore-not-found=$(ignore-not-found) -f 'hack/cnbi-*.yaml'
 
+define managed_operators
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-pipelines-operator
+  namespace: openshift-operators
+spec:
+  channel: latest
+  name: openshift-pipelines-operator-rh
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cert-manager
+  namespace: openshift-operators
+spec:
+  channel: latest
```

Comment:
Similarly to above:

```suggestion
  channel: stable
  installPlanApproval: Manual
  startingCSV: cert-manager.v1.9.1
```

Reply:
No problem for me.
installPlanApproval should be manual for Pipelines as well, should it not ? Otherwise it won't stick to that version.
Btw I think I had some troubles as first using the stable channel for cert-manager, but not sure about that. I might have use the version in cert-manager docs (which had a different source, probably why it was not working)

Reply:
The problem with manual approval is that it requires intervention - that's why I mentioned automatic above... but you are right I think: we can't pin and have it automatic at the same time.

Are we ok with that manual approval?

About stable cert-manager not working: a couple of weeks ago I hit this issue that affected v1.10.0 (from the stable channel) at the time: https://github.com/cert-manager/cert-manager/issues/5551. Maybe you hit that too?  (that's when I pinned my version to v1.9.1)

Reply:
FYI https://docs.openshift.com/container-platform/4.11/cicd/pipelines/op-release-notes.html#release-notes-1-8-1_op-release-notes 

on which version of OpenShift is RHODS based? do we know?

Reply:
anecdotically, from a RHODS cluster: rhods-operator version 1.18.0-9 runs on OCP 4.10 with openshift pipelines 1.6.4